tests/contract/test_resource_client.py

```diff
@@ -710,3 +710,45 @@ def test_assert_primary_identifier_not_updated_fail(resource_client):
             {"a": 1, "b": 2, "c": 3},
             {"a": 1, "b": 2, "c": 4},
         )
+
+
+def test_does_primary_identifier_exist_success(resource_client):
+    schema = {
+        "properties": {
+            "a": {"type": "number", "const": 1},
+            "b": {"type": "number", "const": 2},
+            "c": {"type": "number", "const": 3},
+        },
+        "readOnlyProperties": ["/properties/b"],
+        "createOnlyProperties": ["/properties/c"],
+        "primaryIdentifier": ["/properties/c"],
+    }
```

Comment:
Can we extract these schemas instead of copy-pasting the same few schemas in all these tests?

Reply:
I can set a global variable with this schema and re-use

Reply:
@anshikg [one more at line 409 btw](https://github.com/aws-cloudformation/cloudformation-cli/blob/4f4512064e1a0dc10ec3c27465d458f373dde50c/tests/contract/test_resource_client.py#L409)

Reply:
Updated.